governance/SIG-contributing-template.md

```diff
@@ -0,0 +1,50 @@
+## We welcome contribution
+Describe what kind of issues a new contributor could pick.
+
+[TODO:] A list of standardized labels “good first issue”?, “contribution welcome”?, etc.
+
+## Contributor checklist
+* Comment on an existing ticket or open a new one before coding any PR
+* Wait for a specific approval reply from mantainers and an `approved-for-PR` label
+* Read and Sign the CLA
+* Follow Tensorflow coding style
+* Follow extra repository specific code style
+* Write and execute tests
+* Lint your code
+
+## Environment setup
+How to checkout the code and mount the volume inside a container
+
+
+## Build from source
+How to build the code from source inside a container
+
+## Codestyle 
+Link to Google style guides and any other specific code style
+Linting info and how to configure pre-commit hooks
+
+## IDE
+Any available info for working with specific IDE
+How to use [Codespaces/Devcontainers](https://github.com/microsoft/vscode-dev-containers) if supported
+
+## DOCS
+To contribute to the documentation please follow [DOCS.md]
+
+## TESTS
+How to write and execute tests (bazel, pytest, etc.) or a link to a specific [TEST.md]
+
+## DEBUG
+How to debug your code
+
+## CI
+Quickly describe your CI
+
+## CODEOWNERS
+Describe how codeownership and maintainership in the repo:
+* How to be a Codeowner
+* How to be a Maintainer
```

Comment:
I would like to suggest we consider adding this as a separate file (CALL_FOR_MAINTAINERS.md?) due to the importance of the role.

Reply:
Good, we could put here just a like to an external markdown. 
I don't know I we could open a new thread later about standardizing a little bit the `CALL_FOR_MAINTAINERS` process/roles. 

E.g. Debian has historically a quite regulated [Maintainer process](https://wiki.debian.org/DebianMaintainer). This is just to make a reference example but it is too complex for us.

Reply:
/cc I think that also @mihaimaruseac could be interested in the Codeowner/Maintainer status and process.